### PR TITLE
Add formatting for comments. Add delete for comments.

### PR DESCRIPTION
**Pull request by tin0d0**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
```
Comment (HidyHan) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java:17]: Import what you need.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -82,8 +82,50 @@ function close_comments() {
  * using async to fetch /data sections
  * improves readability and simpleness
  */
-async function getDataAsyncAwait() {
-  const response = await fetch('/data');
-  const comments = await response.text();
-  document.getElementById('data-container').innerText = comments;
+//async function getDataAsyncAwait() {
```
Comment (HidyHan) [inline portfolio/src/main/webapp/script.js]: Remove unused code.

**Review by HidyHan — APPROVED** [review]
Looks good!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (tin0d0) [reply portfolio/src/main/webapp/script.js]: Sounds good! Thank you!

**Review by jileiwang-google — COMMENTED** [review]
LGTM

**Pull request merged**